.github/workflows/main.yml

```diff
@@ -745,7 +745,7 @@ jobs:
     name: Record the result of testing and building steps
     runs-on: ubuntu-latest
     needs:
-      - test_zkasm
+      - test_zkasm_64
```

Comment:
Is the cause for CI failures :)

Reply:
Hmm, strange, while merging I was accepting incoming changes everywhere, how this wasn't changed due to mege?.. But it works, thanks!